homework7.playground/Contents.swift

```diff
@@ -0,0 +1,44 @@
+import UIKit
+
+class School {
+    var students = [Student]()
+    
+    init(students: [Student] = [firstStudent, secondStudent, thirdStudent, fourthStudent]) {
+        self.students = students
+    }
+    func printInfo() {
+        let info = students
+        print(info)
```

Comment:
You need to print all students info. This way you just print the array type (I believe it would print something like "Array<Student>"). You need to call your implemented `printInfo` method for every student here